tests/test_click.py

```diff
@@ -3,6 +3,7 @@
 
 import pytest
 from click.testing import CliRunner
+

```

Comment:
Running `isort` removes this space. Run isort and Python black on the whole file please.

Reply:
`isort` created that empty line. I also just re-ran it on the file and it did not change anything.

Reply:
What version of `isort` are you using? Run `pip install isort --upgrade` and run `isort` on the file again.

Reply:
Tried with `isort` version 5.9.2 and still the same. Even if I mix imports it will still create an empty line. Well I will just remove it manually now.

Reply:
Well, I guess black and isort have some conflicts.